docs/lua-modules/mcp23017.md

```diff
@@ -0,0 +1,230 @@
+# Lua MCP23017 Module for NodeMCU / ESP8266
+
+| Since  | Origin / Contributor  | Maintainer  | Source  |
+| :----- | :-------------------- | :---------- | :------ |
+| 2020-04-10 | [Marcel P.](https://github.com/plomi-net) | [Marcel P.](https://github.com/plomi-net) | [mcp23017.lua](../../lua_modules/mcp23017/mcp23017.lua) |
+
+
+This Lua module provides access to the MCP23017 module.
```

Comment:
this sentence sounds pointless - maybe replace `MCP23017 module` with `MCP23017  I²C I/O Expander` ? I'm not a native speaker but it sounds a little bit better.

Reply:
@galjonsfigur Clearly, i'm not a native speaker too. I have change it.